src/actions/actionTypes.js

```diff
@@ -7,10 +7,12 @@ export const SIGN_IN_SUCCESS = 'SIGN_IN_SUCCESS';
 export const CREATE_TARGET = 'CREATE_TARGET';
 export const LOAD_TARGETS = 'LOAD_TARGETS';
 export const UPDATE_TARGET = 'UPDATE_TARGET';
-export const UPDATE_FREE_TARGET = 'UPDATE_FREE_TARGET';
 export const DELETE_TARGET = 'DELETE_TARGET';
 export const RESET_TARGETS = 'RESET_TARGETS';
+export const UPDATE_FREE_TARGET = 'UPDATE_FREE_TARGET';
 export const RESET_FREE_TARGET = 'RESET_FREE_TARGET';
+export const UPDATE_SELECTED_TARGET = 'UPDATE_SELECTED_TARGET';
```

Comment:
I'm not agree with that approach, in particular for someone that is learning. Here aren't repeated code, are just constants. I know that `mirrorCreator` is going to create the constants for you, but it's really necessary? Also, add a external library to do that, I think is too much. You can write your self `mirrorCreator` function in a helper.

@felipegarcia92 WDYT?